src/app/components/device-list/deviceList.test.js

```diff
@@ -0,0 +1,61 @@
+import {DeviceList} from './deviceList';
+
+describe('Device List', () => {
+    describe('#constructor', () => {
+        it('should be defined', () => {
+            expect(DeviceList).toBeDefined();
```

Comment:
Such cases are needless and were added only for test purpose, so you can remove it